README.md

```diff
@@ -120,6 +123,30 @@ need to be provided in order for this action to successfully run using one of th
 | ------------------------- | -------- | ----------- |
 | `imageToDeploy`           | Yes (for this scenario) | The name of the image that has already been pushed to a registry and will be deployed to the Container App by this action. If this image is found in an ACR instance that requires authentication to pull, the `acrName` argument, or the `acrUsername` and `acrPassword` arguments, can be provided to authenticate requests to the ACR instance. |
 
+### Arguments required for using a YAML configuration file
+
+| Argument name             | Required | Description |
+| ------------------------- | -------- | ----------- |
+| `yamlConfigPath`          | Yes (for this scenario) | Full path (on the executing GitHub runner) to the YAML file detailing the configuration of the Container App. |
+
+#### Important note on the YAML configuration file
+
+When providing the `yamlConfigPath`, identity arguments (_e.g._, `containerAppName`, `resourceGroup`) provided
+to this action will still be used, but non-identity arguments (_e.g._, `containerAppEnvironment`, `environmentVariables`,
+`ingress`, etc.) will be ignored in favor of the values provided in the YAML configuration file. Identity arguments
+provided in the YAML configuration file **are not currently used** and should be provided as arguments to the action.

```

Comment:
I'm wondering if we should say that the logic here works exactly the same as the Container Apps CLI. I'm not saying we should remove anything from your explanation, but it might help people understand that it's not as if we reimplemented everything on our side: here this logic comes directly from the Container Apps side, and that's why we act exactly the same way.

Reply:
Good call -- I just added a snippet at the end of this section to call out that we're using `create` and `update` and that users can refer to those documents for any additional questions that may have about the intended behavior when the YAML configuration file is provided.

Reply:
Thanks!